peps/pep-0750.rst

```diff
@@ -452,11 +518,22 @@ Exceptions raised in t-string literals are the same as those raised in f-string
 literals.
 
 
+No ``Template.__str__()`` Implementation
+----------------------------------------
+
+The ``Template`` type does not provide a specialized ``__str__()`` implementation;
+it inherits the default implementation from the ``object`` class.
+
+This is because ``Template`` instances are intended to be used by template processing
+code, which may return a string or any other type. There is no canonical way to
+convert a Template to a string.
```

Comment:
Is there a repr()? It should return the entire template literal (including `t"` and `"`) as a string, preserving the text between the `{}` -- possibly normalizing cases where the template doesn't preserve the exact text inside those.

Reply:
Ah, good question. I don't see why _not_ to do this!

Reply:
IIRC if there is a repr but no str, str falls back to the repr instead of inheriting from object.